include/image-commands.mk

```diff
@@ -133,6 +133,8 @@ define Build/append-md5sum-ascii-salted
 	rm $@.salted
 endef
 
+UBI_NAND_SIZE_LIMIT= ($(IMAGE_SIZE) - ($(NAND_SIZE) * 20 / 1024))
```

Comment:
The kernel config file says:
```
config MTD_UBI_BEB_LIMIT
	int "Maximum expected bad eraseblock count per 1024 eraseblocks"
```
So you can't use the nand flash size to calculate the reserved block size. Use attached mtd partition size instead.

Reply:
No, it really does work that way. See for example: https://cateee.net/lkddb/web-lkddb/MTD_UBI_BEB_LIMIT.html
```
To put it differently, if this value is 20, UBI will try to reserve about 1.9% of physical
eraseblocks for bad blocks handling. And that will be 1.9% of eraseblocks on the entire
NAND chip, not just the MTD partition UBI attaches. This means that if you have, say,
a NAND flash chip admits maximum 40 bad eraseblocks, and it is split on two MTD
partitions of the same size, UBI will reserve 40 eraseblocks when attaching a partition.
```

I noticed that came as a surprise to many people on mailing lists during my research for this PR...